civis_jupyter_notebooks/assets/civis_client_config.py

```diff
@@ -1,22 +1,12 @@
 import os
 import civis
 from civis_jupyter_notebooks.platform_persistence import logger as LOGGER
-import pip
-
-REQUIREMENTS_PATH = os.path.expanduser(os.path.join('~', 'work', 'requirements.txt'))
 
 if 'CIVIS_API_KEY' in os.environ:
```

Comment:
Mmmmk. I am happy to dump it, but we need to talk to product first. Can you take the lead on that?

Reply:
Sure

Reply:
Why get rid of it? I think it's a nice feature.

Reply:
Too many guard rails. Just give people a notebook. If they want it, they know how to find it. Plus it makes the notebook non-portable. If they download it, then they have to add the import themselves before it will run (assuming they use the client). Simpler is better.

Reply:
Let's leave it in